doc/man/8/rbd.rst

```diff
@@ -523,7 +521,7 @@ Commands
 
 :command:`mirror pool demote` [*pool-name*]
   Demote all primary images within a pool to non-primary.
-  Every mirroring enabled image will demoted in the pool.
+  Every mirror enabled image in the pool will be demoted.
```

Comment:
If replacing "mirroring" with "mirror" here, I think a hyphen is in order:
```suggestion
  Every mirror-enabled image in the pool will be demoted.
```

Reply:
Done.